cmd/nerdctl/image_save.go

```diff
@@ -69,29 +67,56 @@ func processImageSaveOptions(cmd *cobra.Command) (types.ImageSaveOptions, error)
 
 	return types.ImageSaveOptions{
 		GOptions:     globalOptions,
-		Stdout:       cmd.OutOrStdout(),
 		AllPlatforms: allPlatforms,
-		Output:       output,
 		Platform:     platform,
 	}, err
 }
 
-func saveAction(cmd *cobra.Command, args []string) error {
-	if len(args) == 0 {
-		return fmt.Errorf("requires at least 1 argument")
+func determineFileToSave(cmd *cobra.Command) (*os.File, error) {
```

Comment:
maybe just put the logic within `saveAction`, which should be cleaner. Most of the logic are just err check.